pkg/gofr/logging/remotelogger/dynamic_level_logger_test.go

```diff
@@ -114,10 +114,10 @@ func TestDynamicLoggerSuccess(t *testing.T) {
 
 	log := testutil.StdoutOutputForFunc(func() {
 		// Create a new remote logger with the mock server URL
-		rl := New(logging.INFO, mockServer.URL, "1")
+		rl := New(logging.INFO, mockServer.URL, "1") // 1 second is the minimum value
```

Comment:
The signature of New should be refactored to pass a time.Duration

Reply:
This is how we populate the levelFetchInterval in `New` :

```go
if c.Logger == nil {
		c.Logger = remotelogger.New(logging.GetLevelFromString(conf.Get("LOG_LEVEL")), conf.Get("REMOTE_LOG_URL"),
			conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
	}
```

if `New` will accpet time.Duartion then user will need to define the unit

Reply:
changed the signature to accept `time.Duration`